management/events/ccmc/.gitignore

```diff
@@ -0,0 +1 @@
+venv
```

Comment:
I can remove this gitignore file. There's no venv anymore.
I think I ought to create a project level requirements.txt instead of having one for each submodule. It might simplify managing the server.